packages/terra-arrange/examples/index.jsx

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
-import PropsTable from '../../../packages/terra-site/src/PropsTable';
-import Markdown from '../../../packages/terra-site/src/Markdown';
+import PropsTable from '../../../packages/terra-props-table/src/PropsTable';
```

Comment:
In `packages/terra-arrage/package.json`, we want to add terra-props-table and terra-markdown as a devDependency.
```
},
  "devDependencies": {
    "terra-props-table": "0.1.0",
    "terra-markdown": "0.1.0"
  }
```